src/js/config/socket.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url": "wss://www2.binaryws.com/websockets/v3?app_id=1098&l=EN",
+  "url": "wss://www2.binaryws.com/websockets/v3?app_id=1098&l=JA",
```

Comment:
This is for staging environment. we need to make more changes later. so this change is fine for now as it fixes the language issue